helm/hpcc/templates/_helpers.tpl

```diff
@@ -579,3 +579,17 @@ Generate list of available services
   public: {{ $esp.public }}
 {{ end -}}
 {{- end -}}
+
+{{/*
+Add resource object
+Pass in a dictionary with me defined
+*/}}
+{{- define "hpcc.addResources" }}
+{{- if .me  }}
```

Comment:
strictly speaking, should this be checking that the .me dictionary has >0 entries? ( as opposed to e.g. being set to a scalar)

Reply:
It wouldn't cause a problem if it did have 0 items.  Empty resources will just be ignored.